Gopkg.lock

```diff
@@ -488,6 +488,14 @@
   revision = "636bf0302bc95575d69441b25a2603156ffdddf1"
   version = "v1.1.1"
 
+[[projects]]
+  digest = "1:530233672f656641b365f8efb38ed9fba80e420baff2ce87633813ab3755ed6d"
+  name = "github.com/golang/mock"
```

Comment:
I see this is a new dependency of gosnmp, maybe if it were generated into the gosnmp_test package we wouldn't need this testing dep?  I'd like to have someone look into if this is possible before updating gosnmp.

Reply:
I understand that you do not want to include this testing dep, but I am not sure how to fix this.